src/worker/tasks/nonceHealthCheckWorker.ts

```diff
@@ -0,0 +1,165 @@
+import { Job, Processor, Worker } from "bullmq";
+import { Address, getAddress } from "thirdweb";
+import {
+  getUsedBackendWallets,
+  inspectNonce,
+} from "../../db/wallets/walletNonce";
+import { getLastUsedOnchainNonce } from "../../server/routes/admin/nonces";
+import { logger } from "../../utils/logger";
+import { redis } from "../../utils/redis/redis";
+import { NonceHealthCheckQueue } from "../queues/nonceHealthCheckQueue";
+import { logWorkerExceptions } from "../queues/queues";
+
+// Configuration
+
+// Number of consecutive periods to check
+// Checking over multiple periods helps to avoid false positives due to timing issues
+const CHECK_PERIODS = 3;
+
+// Frequency of the worker
+const RUN_FREQUENCY_SECONDS = 20; // Run every 20 seconds
```

Comment:
This used to be every minute? I think every minute still makes sense unless you had a reason to make it run more frequently.